tests-e2e/step-definitions/core/vue.js

```diff
@@ -67,20 +67,17 @@ Given('I go to the evan.network startpage', async () => {
 });
 
 When(/I log out from vue/, async () => {
-  client.useCss();
-  const evan = testUtils.setupEvan(client);
-
-  if (loggedIn) {
-    loggedIn = false;
+  loggedIn = false;
 
-    await client.url(`${evan.baseUrl}#/dashboard.vue.evan`);
-    await client.waitForElementPresent('#evan-dapp-profile', 10 * 1000);
-    await client.click('#evan-dapp-profile');
-    await client.waitForElementPresent('#evan-logout', 10 * 1000);
-    await client.click('#evan-logout');
-    await client.waitForElementPresent('.modal-content #submit-logout', 10 * 1000);
-    await client.click('#submit-logout');
-  }
+  const evan = testUtils.setupEvan(client);
+  const language = global.tags && global.tags.includes('@german') ? 'de' : 'en';
```

Comment:
Prefer an "if" here. This reads like `language` could become a boolean, while the name suggests it being string.

Reply:
Was just copied from before. I can change it

Reply:
Ah OK :)